src/Listener/ViewListener.php

```diff
@@ -120,21 +122,6 @@ public function setFlash(Event $event)
     }
 
     /**
-     * Get the brand name to use in the default template.
-     *
-     * @return string
-     */
-    protected function _getBrand()
```

Comment:
Since a protected method has been removed bump the minor version when making a release.